examples/nlp/language_modeling/conf/megatron_ul2_config.yaml

```diff
@@ -91,22 +69,22 @@ model:
     merge_file: null
     num_sentinel_tokens: 100
 
+  # weight init
+  embedding_init_method_std: 0.02 # Standard deviation of the zero mean normal distribution used for weight initialization.')
+
+  # embedding dropout
+  embedding_dropout: 0.1
+
   # precision
   native_amp_init_scale: 4294967296 # 2 ** 32
```

Comment:
`native_amp_init_scale` and `native_amp_growth_interval` seems to share same values across configs. Perhaps should be moved to base to?

Reply:
The base config is specific to a single `ParallelTransformer` while fp16 scaler-related args are related to the whole model. Other parts of the config like this could potentially go to another shared config file, but that's a discussion for later I think?